cmd/av/branch.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"fmt"
+
+	"emperror.dev/errors"
+	"github.com/aviator-co/av/internal/git"
+	"github.com/aviator-co/av/internal/meta"
+	"github.com/spf13/cobra"
+)
+
+var branchCmd = &cobra.Command{
+	Use:     "branch",
```

Comment:
Sorry. Actually, this should be a part of `av stack branch`.

Reply:
OK, I'll fix it